stretch_core/stretch_core/stretch_driver.py

```diff
@@ -456,152 +456,159 @@ def runstop_service_callback(self, request):
         else:
             self.robot.pimu.runstop_event_reset()
 
-        return SetBoolResponse(
-            success=True,
-            message='is_runstopped: {0}'.format(request.data)
-            )
+        response.success = True,
+        response.message = 'is_runstopped: {0}'.format(request.data)
```

Comment:
Getting a weird Python error that doesn't include a trace, but I think originates from here.

```bash
binitshah@binitshah:~/ament_ws$ ros2 run stretch_core stretch_driver --ros-args --param controller_calibration_file:=/home/binitshah/ament_ws/src/stretch_ros2/stretch_core/config/controller_calibration_head_factory_default.yaml
1633838936.003407 [0] stretch_dr: using network interface wlp2s0 (udp/192.168.1.85) selected arbitrarily from: wlp2s0, docker0
[INFO] [1633838936.130366115] [stretch_driver]: For use with S T R E T C H (TM) RESEARCH EDITION from Hello Robot Inc.
[INFO] [1633838936.130936911] [stretch_driver]: stretch_driver started
[INFO] [1633838936.140086599] [stretch_driver]: mode = position
[INFO] [1633838936.140669652] [stretch_driver]: stretch_driver: Changed to mode = position
[INFO] [1633838936.141533106] [stretch_driver]: broadcast_odom_tf = False
[INFO] [1633838936.142517868] [stretch_driver]: Loading controller calibration parameters for the head from YAML file named /home/binitshah/ament_ws/src/stretch_ros2/stretch_core/config/controller_calibration_head_factory_default.yaml
[INFO] [1633838936.144260116] [stretch_driver]: controller parameters loaded = {'pan_angle_offset': 0.0, 'tilt_angle_offset': 0.0, 'pan_looked_left_offset': 0.0, 'tilt_looking_up_offset': 0.0, 'arm_retracted_offset': 0.0, 'tilt_angle_backlash_transition': -0.4}
[INFO] [1633838936.144794426] [stretch_driver]: self.head_tilt_calibrated_offset_rad in degrees = 0.0
[INFO] [1633838936.145298865] [stretch_driver]: self.head_pan_calibrated_offset_rad in degrees = 0.0
[INFO] [1633838936.145797850] [stretch_driver]: self.head_pan_calibrated_looked_left_offset_rad in degrees = 0.0
[INFO] [1633838936.146315788] [stretch_driver]: self.head_tilt_backlash_transition_angle_rad in degrees = -22.918311805232932
[INFO] [1633838936.146836548] [stretch_driver]: self.head_tilt_calibrated_looking_up_offset_rad in degrees = 0.0
[INFO] [1633838936.147343552] [stretch_driver]: self.wrist_extension_calibrated_retracted_offset_m in meters = 0.0
[INFO] [1633838936.158226150] [stretch_driver]: stretch_driver rate = 15.0 Hz
[INFO] [1633838936.158924776] [stretch_driver]: stretch_driver timeout = 1.0 s
[INFO] [1633838936.160145527] [stretch_driver]: stretch_driver use_fake_mechaduinos = False
[INFO] [1633838936.160880146] [stretch_driver]: stretch_driver base_frame_id = base_link
[INFO] [1633838936.161459389] [stretch_driver]: stretch_driver odom_frame_id = odom
The following exception was never retrieved: The 'data' field must be of type 'bool'
The following exception was never retrieved: The 'data' field must be of type 'bool'
The following exception was never retrieved: The 'data' field must be of type 'bool'
The following exception was never retrieved: The 'data' field must be of type 'bool'
The following exception was never retrieved: The 'data' field must be of type 'bool'
The following exception was never retrieved: The 'data' field must be of type 'bool'
The following exception was never retrieved: The 'data' field must be of type 'bool'
The following exception was never retrieved: The 'data' field must be of type 'bool'
The following exception was never retrieved: The 'data' field must be of type 'bool'
The following exception was never retrieved: The 'data' field must be of type 'bool'
The following exception was never retrieved: The 'data' field must be of type 'bool'
The following exception was never retrieved: The 'data' field must be of type 'bool'
The following exception was never retrieved: The 'data' field must be of type 'bool'
The following exception was never retrieved: The 'data' field must be of type 'bool'
The following exception was never retrieved: The 'data' field must be of type 'bool'
The following exception was never retrieved: The 'data' field must be of type 'bool'
The following exception was never retrieved: The 'data' field must be of type 'bool'
The following exception was never retrieved: The 'data' field must be of type 'bool'
The following exception was never retrieved: The 'data' field must be of type 'bool'
The following exception was never retrieved: The 'data' field must be of type 'bool'
The following exception was never retrieved: The 'data' field must be of type 'bool'
^CCaught signal 2
The following exception was never retrieved: The 'data' field must be of type 'bool'
The following exception was never retrieved: The 'data' field must be of type 'bool'
The following exception was never retrieved: The 'data' field must be of type 'bool'
The following exception was never retrieved: The 'data' field must be of type 'bool'
The following exception was never retrieved: The 'data' field must be of type 'bool'
The following exception was never retrieved: The 'data' field must be of type 'bool'
The following exception was never retrieved: The 'data' field must be of type 'bool'
The following exception was never retrieved: The 'data' field must be of type 'bool'
The following exception was never retrieved: The 'data' field must be of type 'bool'
The following exception was never retrieved: The 'data' field must be of type 'bool'
```

Reply:
Is it possible to select the network interface as a ROS param in ROS2? We could add this to the launch file.

```
1633838936.003407 [0] stretch_dr: using network interface wlp2s0 (udp/192.168.1.85) selected arbitrarily from: wlp2s0, docker0
```